docs/example-scenario/apps/ecommerce-scenario.md

```diff
@@ -0,0 +1,152 @@
+author: masonch
+ms.date: 7/13/18
+---
+# E-Commerce on Azure Platform as a Service
+
+Azure Platform-as-a-Service (PaaS) enables you to deploy enterprise grade e-commerce applications, and lets you adapt to the size and seasonality of your business. When demand for your products or services takes off — predictably or unpredictably — you can be prepared to handle more customers and more transactions automatically. Additionally, take advantage of cloud economics by paying only for the capacity you use. In short, focus on your sales and leave the infrastructure management to your cloud provider.
+
+This document will help you will learn about various Azure PaaS components and considerations used to bring together to deploy a sample e-commerce application, *Relecloud Concerts*, an online concert ticketing platform.
+
+## Potential use cases
+
+Consider this scenario for the following use cases:
+
+* Building an application that needs elastic scale to handle bursts of users at different times.
+* Building an application that is designed to operate at high availability in different Azure regions around the world.
+
+## Architecture
+
+![Sample scenario architecture for an e-commerce application][architecture-diagram]
+
+This scenario covers purchasing tickets from an e-commerce site, the data flows through the scenario as follows:
+
+1. Azure Traffic Manager routes a user's request to the e-commerce site hosted in Azure App Service.
+2. Azure CDN serves static images and content to the user.
+3. User signs in to the application through an Azure Active Directory B2C tenant.
+4. User searches for concerts using Azure Search.
+5. Web site pulls concert details from Azure SQL Database. 
+6. Web site refers to purchased ticket images in Blob Storage.
+7. Database query results are cached in Azure Redis Cache for better performance.
+8. User submits ticket orders and concert reviews which are placed in the queue.
+9. Azure Functions processes order payment and concert reviews.
+10. Cognitive services provide an analysis of the concert review to determine the sentiment (positive or negative).
+11. Application Insights provides performance metrics for monitoring the health of the web application.
+
+### Components
+
+* [Azure CDN][docs-cdn] delivers static, cached content from locations close to users to reduce latency.
+* [Azure Traffic Manager][docs-traffic-manager] controls the distribution of user traffic for service endpoints in different Azure datacenters.
+* [App Services - Web Apps][docs-webapps] hosts web applications allowing auto-scale and high availability without having to manage infrastructure.
+* [Azure Active Directory - B2C][docs-b2c] is an identity management service that enables customization and control over how customers sign up, sign in, and manage their profiles in an application.
+* [Storage Queues][docs-storage-queues] stores large numbers of queue messages that can be accessed by an application.
```

Comment:
Why Storage Queues and not Service Bus?

Reply:
I'll add this in the alternative section